examples/express/index.js

```diff
@@ -0,0 +1,164 @@
+'use strict'
+
+// FOR DEVELOPMENT ONLY: allow for self-signed/invalid certificates universally
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+
+// Use Express.js for routing.
+const express = require('express')
+var session = require('express-session')
+
+const app = express()
+
+app.use(session({
+  secret: 'CHANGE-IT',
+  resave: false,
+  saveUninitialized: true,
+  cookie: {}
+}))
+
+// Import
+const helper = require('openid-client-helper')
+
+// // A ForgeRock configuration example.
+// const issuerMetadata = {
+//   issuer: 'https://default.iam.example.com/am/oauth2',
+//   revocation_endpoint: 'https://default.iam.example.com/am/oauth2/token/revoke'
+// }
+// const clientMetadata = {
+//   client_id: 'node-openid-client',
+//   client_secret: 'password',
+//   token_endpoint_auth_method: 'client_secret_post', // The default is `'client_secret_basic'`.
+//   // response_codes: ['code'], // The first one listed will be used; the default is `['code']`.
+//   redirect_uris: ['http://localhost:3333/redirect'], // The first one listed will be used.
+//   post_logout_redirect_uris: ['http://localhost:3333'] // The first one listed will be used.
```

Comment:
We should change these parameters to use a single string value rather than this "single element array" pattern.

Reply:
Changed.